fastlane/Fastfile

```diff
@@ -88,7 +88,7 @@ platform :android do
     # run the tests, but dont puke everywhere until the report is printed.
     begin
       gradle(
-        task: "vimeo-networking:test example-java-android:build example-kotlin-android:build models:build"
+        task: "vimeo-networking:test example-java-android:build example-kotlin-android:build models:build api:build auth:build"
```

Comment:
should probably use `test` instead of `build` at least for the non-example modules. I don't know that we have any tests yet, but we should future proof it so that we don't check in failing tests.